certified-connectors/Booking Room Pro/Readme.md

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Readme file is not in the correct format. There are some missing sections. Please refer to this [link](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) for a template readme file.